sdk/textanalytics/ai-text-analytics/README.md

```diff
@@ -2,7 +2,7 @@
 
 [Azure TextAnalytics](https://azure.microsoft.com/services/cognitive-services/text-analytics/) is a cloud-based service that provides advanced natural language processing over raw text, and includes six main functions:
 
-__Note:__ This SDK targets Azure Text Analytics service API version 3.0.
+__Note:__ This SDK targets Azure Text Analytics service API version 3.1.0-preview.1.
```

Comment:
are JS still targeting 3.1-preview 1?

Reply:
I will target 3.1.0-preview2 in another PR.